docs/source/deploying/index.md

```diff
@@ -28,3 +28,4 @@ html_meta:
 - [Using apache](./apache.md)
 - [Using sentry](./sentry.md)
 - [Performance improvements](./performance.md)
+- [Non-root deployment](./prefixed-root.md)
```

Comment:
Actually, we should move this entry into the toctree, following that syntax, then remove this list, delete the toctree option `:hidden: true`, and change to `:maxdepth: 1`. This list might have been an artifact from Mkdocs.